### PR TITLE
Fix timeline width

### PR DESCRIPTION
**Pull request by ralfting**

### CODE REVIEW

File: src/components/ScatterPlot/index.js [hunk 1]
```diff
@@ -27,7 +27,12 @@ class ScatterPlot extends Component {
       <div className="scatter-plot">
         <div className="topics">{ topics }</div>
         <DetailsBubble close={ () => this.props.closeBubble() }/>
-        <ScatterPlotTimeline domain={ domain } />
+        <div className="scatter-row ">
```
Comment (ghost) [inline src/components/ScatterPlot/index.js]: Please, remove the trailing space.

File: src/components/ScatterPlotTimeline/styles.scss [hunk 2]
```diff
@@ -1,6 +1,7 @@
 @import '../../config';
 
 .timeline {
+width:100%;
```
Comment (ghost) [inline src/components/ScatterPlotTimeline/styles.scss]: Use proper indentation.

**Review by ghost — CHANGES_REQUESTED** [review]
Minor changes, but nonetheless important.

File: src/components/ScatterPlotTimeline/styles.scss (hunk 2 above)
Comment (ralfting) [reply src/components/ScatterPlotTimeline/styles.scss]: Fixed!

**Review by anapaulavnoronha — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**